### PR TITLE
docs(helpers,types): add discord doc links & remove leftover comments

### PR DESCRIPTION
**Pull request by lts20050703**
# Check #2219 first!

### CODE REVIEW

**Review by itohatweb — CHANGES_REQUESTED** [review]
This PR does much more than what the title indicates

**Comment by lts20050703** [comment]
sorry about that

File: helpers/discovery/editDiscovery.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export async function editDiscovery(bot: Bot, guildId: bigint, data: ModifyGuild
 }
 
 // TODO: add docs link
+/** https://github.com/discord/discord-api-docs/pull/1762 */
```
Comment (itohatweb) [inline helpers/discovery/editDiscovery.ts:33]: Please remove the `TODO` comment

File: types/discord.ts [hunk 2]
```diff
@@ -2161,6 +2160,7 @@ export interface DiscordMessageReactionRemoveAll
   extends Pick<DiscordMessageReactionAdd, "channel_id" | "message_id" | "guild_id"> {}
 
 // TODO: add docs link
+/** https://github.com/discord/discord-api-docs/pull/1762 */
```
Comment (itohatweb) [inline types/discord.ts:2163]: Please remove the `TODO` comment.

**Review by itohatweb — CHANGES_REQUESTED** [review]

File: helpers/discovery/editDiscovery.ts (hunk 1 above)
Comment (lts20050703) [reply helpers/discovery/editDiscovery.ts:33]: @itohatweb check out https://github.com/discord/discord-api-docs/pull/1762#issuecomment-1082095053
the document discovery pr is closed not merged
I just want to put the link to the PR so the next person wont have to waste half an hour searching discord docs and cant find discovery stuff

Comment (itohatweb) [reply helpers/discovery/editDiscovery.ts:33]: Then remove this reference.

**Review by itohatweb — APPROVED** [review]

**Pull request merged**